### ISSUE TITLE
Loss of ambulation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7191]
**Preferred term label:**
Loss of ambulation

**Synonyms**
Wheelchair dependency (RELATED)

**Definition (free text, please give PubMed ID)**
Said of a person who loses a previously existant ability to walk and often associated with dependency on a wheelchair for moving from place to place.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619518
PMID:32403198
5/11, juvenile onset

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Can this be merged with [Progressive inability to walk HP:0002505](https://hpo.jax.org/app/browse/term/HP:0002505)?

**Comment by pnrobinson** [comment]
I think this is a synonym
